src/pages/Graph/GraphToolbar/GraphSettings.tsx

```diff
@@ -280,7 +280,15 @@ class GraphSettings extends React.PureComponent<GraphSettingsProps, GraphSetting
         id: 'filterSecurity',
         labelText: 'Security',
         isChecked: showSecurity,
-        onChange: toggleGraphSecurity
+        onChange: toggleGraphSecurity,
+        tooltip: (
+          <div style={{ textAlign: 'left' }}>
+            <div>
+              Show closed or open lock icons on edges that disagree with the global mTLS policy. Note that the global
```

Comment:
"disagree" doesn't seem to be the right word here. Makes it seem like the value displayed is wrong or it should be something else.

Perhaps "contrasts" or "differs from"?

```suggestion
              Show closed or open lock icons on edges that differ from the global mTLS policy. Note that the global
```